charts/brig/templates/configmap.yaml

```diff
@@ -87,18 +87,34 @@ data:
           {{- end }}
 
       user:
+        {{- if .emailSMS.user }}
+        activationUrl: {{ .emailSMS.user.activationUrl }}
+        smsActivationUrl: {{ .emailSMS.user.smsActivationUrl }}
+        passwordResetUrl: {{ .emailSMS.user.passwordResetUrl }}
+        invitationUrl: {{ .emailSMS.user.invitationUrl }}
+        deletionUrl: {{ .emailSMS.user.deletionUrl }}
+        {{- else }}
         activationUrl: {{ .externalUrls.nginz }}/activate?key=${key}&code=${code}
         smsActivationUrl: {{ .externalUrls.nginz }}/v/${code}
         passwordResetUrl: {{ .externalUrls.nginz }}/password-reset/${key}?code=${code}
         invitationUrl: {{ .externalUrls.nginz }}/register?invitation_code=${code}
         deletionUrl: {{ .externalUrls.nginz }}/users/delete?key=${key}&code=${code}
+        {{- end }}
 
       provider:
+        {{- if .emailSMS.provider }}
```

Comment:
Can you de-dent the 'if' and 'else' clauses like they do on line 104? It's bike-shedding, but it makes it a little easier to see where the control flow is happening.

Reply:
👍